gcloud-java-dns/src/main/java/com/google/gcloud/dns/spi/DefaultDnsRpc.java

```diff
@@ -54,6 +62,26 @@ public DefaultDnsRpc(DnsOptions options) {
   }
 
   @Override
+  public BatchRequest enqueueListZones(BatchRequest batch, final DnsBatch.Request request,
```

Comment:
This seems to me like a utility conversion methods that should reside in `DnsImpl` rather than a public method for `DnsRpc`. I see however that [queue](https://developers.google.com/api-client-library/java/google-api-java-client/reference/1.19.1/com/google/api/client/googleapis/services/json/AbstractGoogleJsonClientRequest#queue(com.google.api.client.googleapis.batch.BatchRequest,%20com.google.api.client.googleapis.batch.json.JsonBatchCallback)) can throw `IOException`. However I think that this `IOException` should not contain any DNS-related error as no network comunication should happen at this stage, right?

I am wondering wether we should expose this method or rather hide it in `DnsImpl`.

Also, is the `request` parameter used?